commercial/app/views/hosted/guardianHostedGallery.scala.html

```diff
@@ -10,34 +11,27 @@
                 <div class="js-hosted-gallery-container hosted-gallery__gallery-container">
                     <div class="js-hosted-gallery-images hosted-gallery__images-container">
                         @guardianHostedGalleryImage(page.images.head, {
+                            <div class="hosted-gallery__image-gradient"></div>
```

Comment:
Hmm, I see. Can you add a class to `hosted-gallery__cta` that adds the gradient?